styles.css

```diff
@@ -0,0 +1,264 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  //display: flex;
+  //background-color: #ff8c00;
+}
+
+body {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  line-height: 1.5;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+#header {
+  display: flex;
+}
+
+#nav-bar {
+  position: fixed;
+  top: 0px;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  background-color: rgb(255, 255, 255);
+  overflow: hidden;
+}
+/*
+@media only screen and (max-width:600px){
+#nav-bar ul {
+  display: flex;
+}
+#nav-bar ul li {
+  padding: 1rem 2rem;
+}

```

Comment:
Please remove out commented code in your codebase. This makes code unclean and brings confusion to other programmers or you in the future.